OsmAnd/src/net/osmand/plus/plugins/srtm/SRTMPlugin.java

```diff
@@ -507,7 +507,7 @@ public boolean onContextMenuClick(@Nullable OnDataChangeUiAdapter uiAdapter, @Nu
 		boolean terrainEnabled = TERRAIN.get();
 		TerrainMode terrainMode = TERRAIN_MODE.get();
 		adapter.addItem(new ContextMenuItem(TERRAIN_ID)
-				.setTitleId(R.string.shared_string_terrain, mapActivity)
+				.setTitleId(R.string.shared_string_hillshade_slope, mapActivity)
```

Comment:
That would be like a duplicate line cause 2nd line is the same.

And I don't like it cause I also want in future hypsometric maps

Reply:
No, below are the respective screenshots: Second line shows which one is selected or and selected status.

For "Hillshade / Slope" you can either select "Hillshade" or "Slope". So far we had called "Hillshade / Slope" "Terrain" in Configure map, but "Hillshade" and "Slope" on the Download UI (download types).

I think now it is appropriate we use Terrain as the overall header for all (currently max. 4) related map types and functionalities), and use the specific "Hillshade / Slope" for the sub-item, in line the respective download types, removing confusion which download is needed for which function? Keep in mind, the idea is to remove the word "Terrain" on any particular sub-item, use it as the overall section headline for our new overall functinality:


![Screenshot_20230727-142534_OsmAnd~](https://github.com/osmandapp/OsmAnd/assets/845735/ccd769b6-5701-487d-845b-c9a6f8ec710d) ![Screenshot_20230727-142544_OsmAnd~](https://github.com/osmandapp/OsmAnd/assets/845735/350252e2-d768-4066-9878-68946d78e187)

Reply:
I think @dmpr0 is right. It should stay - https://docs.qgis.org/3.28/en/docs/training_manual/rasters/terrain_analysis.html (Terrain), here it says Raster Terrain. We don't plan to have any vector terrain there.

So it's better to change the card name to avoid name duplication or not change it.

Reply:
@vshcherb, @dmpr0 But there's no contradiction in the link above at all? Also they use "Terrain" as the super-category for hillshade and other methods, exactly like U propose? Why would we not call our Hillshade and Slope maps consistently "Hillshade" and "Slope", and the issue is solved?

Reply:
@dmpr0 @vshcherb   PS: In light of me now understanding our situation a bit better: Are we back to perhaps leaving things as they are now in v4.5.8, only simply renaming the section **headline**, perhaps to "Topography"?